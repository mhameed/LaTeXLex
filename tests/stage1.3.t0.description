author: mmh
date: 09/02/28
description: 
basic test to check prebrace, and postbrace, in normal text and in verbatims.
