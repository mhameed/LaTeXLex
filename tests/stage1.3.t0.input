Here comes some {\it italic} text.
Here comes some{\it italic} text.
{\it italic}
an \it{italicised line of some length}
an\it{ italicised line of some length}
an\it{imal}
the rest \it of this line is italicised
\it{italicised line of some length}
\begin{verbatim}
\it{this is a comment.}
Here comes a {\it italic} text.
\end{verbatim}
an {\it italicised line of some length}
an{\it italicised line of some length}
an{\it imal}
