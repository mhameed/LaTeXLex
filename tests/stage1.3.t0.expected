Here comes some italic text.
Here comes someitalic text.
italic
an italicised line of some length
an italicised line of some length
animal
the rest of this line is italicised
italicised line of some length
\begin{verbatim}
\it{this is a comment.}
Here comes a {\it italic} text.
\end{verbatim}
an italicised line of some length
anitalicised line of some length
animal
